### PR TITLE
Step-by-step guide to distribute a new release (PR)

### PR DESCRIPTION
**Pull request by deyanzz**
**Description**:

This PR introduces a short `CONTRIBUTING.md` guide with few steps describing how to prepare and distribute a new release version of the `C++ SDK`.

**Related issue(s)**:

**Fixes**: [Issue #222 - Step-by-step guide to distribute a new release](https://github.com/hashgraph/hedera-sdk-cpp/issues/222)

**Notes for reviewer**:

Any suggestions for improvements will be appreciated.

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Contributing to the Hedera SDK for JavaScript
```
Comment (rwalworth) [inline CONTRIBUTING.md:1]:
```suggestion
# Contributing to the Hedera SDK for C++
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+It will populate the newly added tag in `origin`. 
+
+### Step 2 - Draft Release in GitHub
+
+Open [Hedera C++ SDK](https://github.com/hashgraph/hedera-sdk-cpp) and search for `Create a new release` link.
+Select tag `v0.3.0` from the dropdown list. Enter the Release title to be same as the release number which is been prepared (for example: `v0.3.0`).
+
+### Step 3 - Notes
+
+It's a good decision to tap direclty on button `Generate release notes` to get an automatically generated notes which can be used as a basis.
```
Comment (rwalworth) [inline CONTRIBUTING.md:32]:
```suggestion
It's a good decision to tap directly on button `Generate release notes` to get automatically generated notes which can be used as a basis.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+#### What's Changed
```
Comment (rwalworth) [inline CONTRIBUTING.md:34]: We currently have this section called `What's New`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+In this section should be filled all of the issues which have been completed and merged into `main` branch. Use only the title of each issue and add a prefix with ➡️.
```
Comment (rwalworth) [inline CONTRIBUTING.md:36]:
```suggestion
This section should be filled with all of the issues which have been completed and merged into `main` branch. Use only the title of each issue and add a prefix with ➡️.
```

This section also should just contain the over-arching new features and bug fixes in plain English.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+#### Examples
+
+In this section should be listed all of the newly added examples. Each of them should represent a link to the implementation (for example: [TransferTokensExample.cc](TransferTokensExample.cc)). It's a good idea also to add prefix with ➡️ before each line.
```
Comment (rwalworth) [inline CONTRIBUTING.md:40]:
```suggestion
This section should list all of the newly added examples. Each of them should represent a link to the implementation (for example: [TransferTokensExample.cc](TransferTokensExample.cc)). It's a good idea also to add prefix with ➡️ before each line.
```

**Review by rwalworth — APPROVED** [review]
Just a few grammatical fixes for the most part

File: CONTRIBUTING.md (hunk 1 above)
Comment (deyanzz) [reply CONTRIBUTING.md]: Fixed in ad3ea037429a4f2cf9cadc72d920c2cf9d020b39.

File: CONTRIBUTING.md (hunk 3 above)
Comment (deyanzz) [reply CONTRIBUTING.md]: Fixed in ad3ea037429a4f2cf9cadc72d920c2cf9d020b39.

**Review by rwalworth — APPROVED** [review]
LGTM
